srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -167,6 +142,48 @@ public Patient getPatient(final int dfn)
         return Patient.Gender.Unknown;
     }
     
+    private void retrieveVitals(final Patient patient)
+    {
+        try
+        {
+            final List<String> vitalResults = fProcedureCaller.doRpc(
+                    fDuz, RemoteProcedure.GMV_LATEST_VM, String.valueOf(patient.getDfn()));
+            // Patient vitals information (including but not limited to BMI, height,
+            // weight, weight 6 months ago). If there are no results, a single line with
+            // an error message is returned.
+            LOGGER.debug("Patient Vital Results: {}", vitalResults);
+            if (vitalResults.size() > 1)
+            {
+                // Parse the returned data and put it into the patient data
+                // This will include the most recent height, current weight, and BMI
+                parseRecentVitalResults(patient, vitalResults);
```

Comment:
I can't put the comment on the line, but you should also log failed regex matches.